includes/Modules/Thank_With_Google/Settings.php

```diff
@@ -69,4 +69,103 @@ protected function get_default() {
 			'buttonPostTypes' => array( 'post' ),
 		);
 	}
+
+	/**
+	 * Gets the setting value.
+	 *
+	 * @since m.e.x.t
```

Comment:
Please be sure to use the correct placeholder as search/replace will miss it otherwise.
```suggestion
	 * @since n.e.x.t
```

Reply:
Oh my goodness, they looks same to me on monospaced font. Sorry!

Reply:
I missed it too, waiting for someone to make a `m.e.x.t` release in ZenHub just to really mess with us 😉